pkg/controllers/cluster/taint_manager.go

```diff
@@ -291,7 +291,7 @@ func (tc *NoExecuteTaintManager) needEviction(clusterName string, annotations ma
 // SetupWithManager creates a controller and register to controller manager.
 func (tc *NoExecuteTaintManager) SetupWithManager(mgr controllerruntime.Manager) error {
 	return utilerrors.NewAggregate([]error{
-		controllerruntime.NewControllerManagedBy(mgr).For(&clusterv1alpha1.Cluster{}).Complete(tc),
+		controllerruntime.NewControllerManagedBy(mgr).Named(TaintManagerName).For(&clusterv1alpha1.Cluster{}).Complete(tc),
```

Comment:
Please comment on the const like the others.
https://github.com/karmada-io/karmada/pull/5799/files#diff-67bee5e7ec3a123c6b42ab16c716b1a180a0c5fc46c7c65899377727c3df3499L42